src/ui/ResponsiveComponent.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import PropTypes from "prop-types";
```

Comment:
This is not a `devDependencies`. It's a `dependencies`. Actually, I'd rather see this (and `react`) in `peerDependencies` as this will be used within apps that already load these packages.